simpleclient/src/main/java/io/prometheus/client/Summary.java

```diff
@@ -137,15 +168,15 @@ public double observeDuration() {
    * {@link SimpleCollector#remove} or {@link SimpleCollector#clear}.
    */
   public static class Child {
-    public static class Value {
-      public final double count;
-      public final double sum;
-      public final CKMSQuantiles quantiles;
+    private static class Value {
```

Comment:
I don't really understand what `Value` is used for outside of the `Summary` class. Should it be immutable? The `final double` values are immutable, but in order to make quantiles immutable, I would need to implement a wrapper that exposes only the `get(double quantile)` method but not the `insert(double value)` method. Is `Value` supposed to be used outside of the `Summary` class for read-only access to the metrics?